internal/pkg/manifest/validate.go

```diff
@@ -49,6 +51,8 @@ var (
 
 	essentialContainerDependsOnValidStatuses = []string{dependsOnStart, dependsOnHealthy}
 	dependsOnValidStatuses                   = []string{dependsOnStart, dependsOnComplete, dependsOnSuccess, dependsOnHealthy}
+	nlbValidProtocols                        = []string{TCP, tls}
```

Comment:
Why not `TLS`?

Reply:
`TCP` is exported while `tls` doesn't need to be.